package.json

```diff
@@ -97,6 +97,7 @@
     "smooth-release": "^8.0.0",
     "style-loader": "^0.17.0",
     "stylelint-webpack-plugin": "^0.7.0",
+    "typelevel-ts": "^0.1.2",
```

Comment:
I've put it in `devDeps` cause it's like this in *alinity-pro*, but maybe in this case should be a `dep`